commands/update/cluster/command_test.go

```diff
@@ -302,3 +336,43 @@ func TestExecuteWithError(t *testing.T) {
 		})
 	}
 }
+
+func Test_modifyClusterLabelsRequestFromArguments(t *testing.T) {
```

Comment:
I would like to use tabletests more often but understand that it can be cumbersome here. So unit tests is still be than no unit tests.